### DISCUSSION TITLE
Issue with issuing new CSR

### DISCUSSION THREAD

**Discussion by jp-mw-team** [openssl/openssl#24733] [Q&A] [answered]
We received below error when creating CSR using openssl. Please let us know the issue as certificate is going ot expire in 2 days.

9C0B0000:error:07000065:configuration file routines:def_load_bio:missing equal sign:crypto\conf\conf_def.c:530:HERE-->43line36
Looking forward for your response regarding this issue.

Thank you.

**Comment by t8m** [comment, marked as answer]
It seems the OpenSSL configuration file you are using is corrupted. You can try to find the `43line36` text in it and fix it.

**Reply by jp-mw-team** [reply]
Hi @t8m ,

I checked the configuration, and the line seems fine. The file was only changed everytime the certificate is being renewed. Recently, there was an upgrade happened on Apache version which might cause the issue probably? Coz before this was upgraded, I believe we were able to issue a CSR. Is there any configuration needed to be updated on openssl side?

**Reply by t8m** [reply]
How does the line look like?
